build.gradle

```diff
@@ -23,22 +23,21 @@ repositories {
 }
 
 dependencies {
-    compile gradleApi()
-    compile('org.codehaus.groovy.modules.http-builder:http-builder:0.7.2') {
+    implementation gradleApi()
+    implementation('org.codehaus.groovy.modules.http-builder:http-builder:0.7.2') {
         exclude(module: 'groovy')
     }
-    compile('org.apache.maven.resolver:maven-resolver-ant-tasks:1.2.0')
-    // provided by Gradle
-    compileOnly('org.apache.maven:maven-project:2.0.11')
-    testRuntime('org.apache.maven:maven-project:2.0.11')
-    testCompile('org.spockframework:spock-core:0.7-groovy-2.0') {
+    implementation('org.apache.maven.resolver:maven-resolver-ant-tasks:1.2.0')
+    implementation('org.apache.maven:maven-core:3.0.5')
+
+    testImplementation('org.spockframework:spock-core:0.7-groovy-2.0') {
```

Comment:
It might be a good idea to update this dependency to a version that closely matches the Groovy version provided by `gradleApi()`, after all Spock is very sensitive to Groovy AST changes